src/models/Category.ts

```diff
@@ -0,0 +1,60 @@
+import type { PopulatedDoc, Types, Document, Model } from "mongoose";
```

Comment:
create a compound unique constraint on `category.organzationId` and `category.name`(category.category here)